templates/schedule.html

```diff
@@ -22,31 +26,17 @@ <h1>Get your tickets now</h1>
         </div>
     </div>
     <div id="programme">
-        <div class="content-wrapper">
-            <div class="content">
-                <h2>Schedule</h2>
-                <table>
-                    <tr>
-                        <td>Thursday 19, September</td>
-                        <td>BarCamp</td>
```

Comment:
we might wanna keep this info. particularly the barcamp is entirely lost with the schedule embed

Reply:
It's lost in the schedule embed, but I'd rather add it in pretalx than to add the table